__tests__/endpoints.test.js

```diff
@@ -284,14 +285,25 @@ describe("PATCH: /api/articles/:article_id ", () => {
 				expect(body).toMatchObject({ msg: "Bad request" });
 			});
 	});
-	test('PATCH: 400 client sends not a number as the article_id', () => {
+	test("PATCH: 400 client sends not a number as the article_id", () => {
 		return request(app)
 			.patch("/api/articles/notanum")
 			.send({ inc_votes: 10 })
 			.expect(400)
-			.then(({body}) => {
-				expect(body).toEqual({"msg": "Bad request"})
+			.then(({ body }) => {
+				expect(body).toEqual({ msg: "Bad request" });
 			});
-		
+	});
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+	test("DELETE: 204 returns no content", () => {
+		return request(app).delete("/api/comments/1").expect(204);
+	});
+	test("DELETE: 400 returns error if invalid number comment ID", () => {
```

Comment:
These 2 tests are mixed up, `404` would be for a legitimate id that is not found whereas `400` would be for an invalid id